src/components/Image/types.ts

```diff
@@ -0,0 +1,38 @@
+import { TPAComponentProps } from '../../types';
+
+export interface ImageProps extends TPAComponentProps {
+  /** The source could be any absolute full URL or a relative URI of a media platform item */
+  src?: string;
+  /** The intrinsic width of the image in pixels */
+  width?: number;
+  /** The intrinsic height of the image in pixels */
+  height?: number;
+  /** The alternative text description of the image. Allowing better SEO when the image has meaning of content */
+  alt?: string;
+  /** A callback to be called when the image is loaded */
+  onLoad?: React.EventHandler<React.SyntheticEvent>;
+  /** A callback to be called if error occurs while loading */
+  onError?: React.EventHandler<React.SyntheticEvent>;
+  /** Specifies how the image is resized to fit its container */
+  resize?: 'contain' | 'cover';
```

Comment:
```suggestion
  resize?: ResizeOptions;
```

Reply:
I wanted to do so but then the story page doesn't show the actual options, meaning only the enum name:
![image](https://user-images.githubusercontent.com/7141972/105130859-75352480-5af0-11eb-9094-cd1f8d0f16f4.png)

Also, it enforces the consumer to use the enum:
![image](https://user-images.githubusercontent.com/7141972/105130959-aa417700-5af0-11eb-84b4-7785e40583c0.png)

Not sure how comfortable it is...